xlive/Blam/Engine/interface/screens/screens_patches.cpp

```diff
@@ -1,10 +1,15 @@
 #include "stdafx.h"
 #include "screens_patches.h"
 
+#include "screen_esrb_warning.h"
 #include "screen_xbox_live_task_progress_dialog.h"
 
 void screens_apply_patches_on_map_load()
 {
 	c_xbox_live_task_progress_menu::apply_patches();
+}
 
+void screens_apply_hooks()
+{
+	c_screen_esrb_warning::apply_patches();
```

Comment:
None of the codes inside `c_screen_esrb_warning::apply_patches();`  are related to `hooking` so maybe rename this function as `screens_apply_patches_on_startup` ?

Reply:
changed the function name to screens_apply_patches